storage-proofs/parameters.json

```diff
@@ -0,0 +1 @@
+../filecoin-proofs/parameters.json
```

Comment:
I think we want the file to live in `storage_proofs`, with the symlink in `filecoin_proofs`. The idea is that there should only be dependencies from `filecoin_proofs` on `storage_proofs` and not vice versa. Does that match your understanding, @cryptonemo ?

Reply:
That's how this should be, or at least was.  I had git mv'd it over, then added a symlink there.

Reply:
It does seem backwards.  That was the intent anyway, will try again to fix that.